packages/cli/test/integration/generators/datasource.integration.js

```diff
@@ -0,0 +1,186 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+// Imports
+const path = require('path');
+const assert = require('yeoman-assert');
+const testlab = require('@loopback/testlab');
+
+const expect = testlab.expect;
+const TestSandbox = testlab.TestSandbox;
```

Comment:
As a side, what's preventing us from using the `import _ from _` pattern here?

Reply:
This is a `JS` project, not a `TS` project which is where the `import _ from _`. Support for this will come to JS eventually -- current status is Unstable as documented here: https://nodejs.org/api/esm.html